### PR TITLE
Improve on delegation logging

### PR DESCRIPTION
**Pull request by paweljakubas**
# Issue Number

#1376 

# Overview

- [x] I have updated how delegation is logged
- [x] I have formtatted UTXTime in wallet creation, ie., cut milliseconds 

# Comments

### CODE REVIEW

**Comment by paweljakubas** [comment]
now : 
```
[cardano-wallet.wallet-engine:Info:66] [2020-02-26 14:09:43.07 UTC] 2d4cc31a: Faucet Wallet, created at 2020-02-26 14:09:39 UTC, delegating to ∅
[cardano-wallet.wallet-engine:Info:66] [2020-02-26 14:10:03.35 UTC] 2d4cc31a: Faucet Wallet, created at 2020-02-26 14:09:39 UTC, delegating to ∅ → 1326209
```

File: lib/core/src/Cardano/Wallet/Primitive/Types.hs [hunk 1]
```diff
@@ -381,15 +383,13 @@ instance NFData WalletDelegation
...
-        "current wallet delegation: " <> build act <> ", awaiting " <> build n
+        "delegating to " <> build act <> " → " <> build n
     build (WalletDelegation act [n1, n2]) =
-        "current wallet delegation: " <> build act <> ", awaiting first " <>
-        build n1 <> " then awaiting " <> build n2
+        "delegating to " <> build act <> " → " <> build n1 <> " → " <> build n2
     build (WalletDelegation act _) =
-        "current wallet delegation: " <> build act <>
-        ", something wrong with awaiting"
+        "delegating to " <> build act <> " something wrong with what's next"
```
Comment (KtorZ) [inline lib/core/src/Cardano/Wallet/Primitive/Types.hs]: Alternatively:

```hs
instance Buildable WalletDelegation where
    build (WalletDelegation act []) =
        "delegating to " <> build 
    build (WalletDelegation act xs) =
         build (WalletDelegation act []) <> " → " <> build (T.intercalate " → " $ build <$> xs)
```

- :heavy_check_mark: avoid repetition of the literal string
- :heavy_check_mark: handles next uniformly

**Review by KtorZ — APPROVED** [review]
LGTM. Minor style suggestion to avoid repetitions.

File: lib/core/src/Cardano/Wallet/Primitive/Types.hs (hunk 1 above)
Comment (paweljakubas) [reply lib/core/src/Cardano/Wallet/Primitive/Types.hs]: in the end I have this : 
```
385 instance Buildable WalletDelegation where                                                                                                                                                 
386     build (WalletDelegation act []) =                                                                                                                                                     
387         "delegating to " <> build act                                                                                                                                                     
388     build (WalletDelegation act xs) =                                                                                                                                                     
389         build (WalletDelegation act [])                                                                                                                                                   
390         <> " → "                                                                                                                                                                          
391         <> build (T.intercalate " → " $                                                                                                                                                   
392                   (LT.toStrict . Builder.toLazyText . build) <$> xs)
```

Comment (paweljakubas) [reply lib/core/src/Cardano/Wallet/Primitive/Types.hs]: T.intercalate accepts [Text] as second argument, not [Builder]

Comment (KtorZ) [reply lib/core/src/Cardano/Wallet/Primitive/Types.hs]: Huhu, good point. So, you could replace `LT.toStrict . Builder.toLazyText . build` with `pretty` (https://hackage.haskell.org/package/fmt-0.6.1.2/docs/Fmt.html#v:pretty) :+1:

Comment (paweljakubas) [reply lib/core/src/Cardano/Wallet/Primitive/Types.hs]: indeed! thanks

**Pull request merged**